conf/salt/project/worker/default.sls

```diff
@@ -24,7 +24,7 @@ include:
         directory: "{{ vars.source_dir }}"
         name: "celery-{{ instance }}"
         command: "worker"
-        flags: "--loglevel=INFO -Q celery --concurrency={{ grains['num_cpus'] * 4 + 1 }}"
+        flags: "--loglevel=INFO -Q queue_{{ instance }} --concurrency=2"
```

Comment:
The command is "worker".  The word "celery" in the previous version of this line was the previous queue name.